modules/pubCommonIdSystem.js

```diff
@@ -161,6 +161,18 @@ export const pubCommonIdSubmodule = {
    */
   name: MODULE_NAME,
 
+  /**
+   * Vendor id of Prebid
+   * @type {Number}
+   */
+  gvlid: GVLID,
+  
+  /**
+   * Vendor id of prebid
+   * @type {Number}
+   */
+  gvlid: GVLID,
+  
```

Comment:
why do you have this twice?

Reply:
honest mistake :(

Reply:
i guess we'll let it slide this time ;-)